integration/cri-o/crio_skip_tests.sh

```diff
@@ -27,6 +65,94 @@ declare -A skipCRIOTests=(
 ['test "annotations passed through"']='This is not working'
```

Comment:
Due to the changes brought by #4434, we will need separated associative arrays during rebase (e.g.,  cgroups_skipCRIOTests, ctr_skipCRIOTests, ...) as per @littlejawa comment.